alchemy/src/cloudflare/worker.ts

```diff
@@ -1191,7 +1199,7 @@ export const _Worker = Resource(
     }
 
     let putWorkerResult: PutWorkerResult;
-    if (dev.type === "remote") {
+    if (watch) {
```

Comment:
wait why is this here? isn't watch only for watching resource changes?

Reply:
That's what this does. It watches for changes to the worker bundle